packages/beacon-node/src/chain/blocks/importBlock.ts

```diff
@@ -146,7 +147,7 @@ export async function importBlock(
 
     for (const attestation of attestations) {
       try {
-        const indexedAttestation = postState.epochCtx.getIndexedAttestation(attestation);
+        const indexedAttestation = postState.epochCtx.getIndexedAttestation(fork, attestation);
```

Comment:
add `TODO: figure out how to reuse the attesting indices computed from state transition`